src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+$cell-side: 100px;
+$cell-sideAndGap: $cell-side + 1;
+$weekDays: ("mon", "tue", "wed", "thu", "fri", "sat", "sun");
+
+.calendar {
+  &__list {
+    display: flex;
+    gap: 1px;
+    padding: 10px;
+    margin: 0;
+    box-sizing: border-box;
+    list-style: none;
+    flex-wrap: wrap;
+    width: $cell-side * 7 + 20px + 6px;
+
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $cell-side;
+    height: $cell-side;
+    border: solid 1px #000;
+    box-sizing: border-box;
+    transition: transform, background-color, 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+
+  $counter: 0;
```

Comment:
It is better to keep all the variables together and not spread them over the file.

It is also necessary to give more meaningful names to variables, for example `daysCounter`.

Reply:
This direct variable refers only to the loop under it, so it might cause some misunderstanding if I move it upper, shall I do it?

Reply:
@rostyslav48 It is worth giving a correct name for this variable and then there will be no misunderstandings

Reply:
Ok, but may I leave it on the same place?